### PR TITLE
Features/enhanced readme

### PR DESCRIPTION
**Pull request by mayaalsamarrai**
Please review and give feedback and comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,129 @@
...
-* inheritance `Tags` between catalogs objects depend on catalog taxonomy (categories inherit this property from parent catalog or category, products inherit this property from parent category)
-* allow using `User groups` from customer profile to products or categories filtration in the storefront search
-* the two different policies for `Tags` propagation in catalog objects hierarchy 
-    - `UpTree` propagate the tags from descendants to parents up the hierarchy
-    ![image](https://user-images.githubusercontent.com/7566324/62931481-ba630c00-bdbe-11e9-9cdf-6d05e955721b.png)
-    
-    
-    - `DownTree` inherites the tags by descendants from their parrent down the hierarchy
-    ![image](https://user-images.githubusercontent.com/7566324/62931421-a3241e80-bdbe-11e9-8f02-fd22d0fbcc6f.png)
+The main concept of VC Personalization module is to personalize the issuing of catalog, price lists and marketing actions using Tags and User Groups.
```
Comment (yecli) [inline README.md]: Maybe rephrase, I do not clearly understand the purpose of the module from this sentense

File: README.md [hunk 2]
```diff
@@ -1,55 +1,129 @@
...
+The Contacts should be first included into  specific User Groups and then assign  catalogs, price lists or promotions to this UG.
 
-### Example:
+## Key features
 
-<details><summary>Managing catalog tags</summary><p>
-  
-![catalog-tags](https://user-images.githubusercontent.com/7536694/31497152-6510aac0-af5e-11e7-9e99-8f3f2d45cca3.gif) </p></details>
+1. Control the visibility of catalog objects such as Product and Categories, through manual tagging of these objects with special property `Tags` which can contain multiple predefined values which also may be defined for customer profile
+1. Inheritance `Tags` between catalogs objects depend on catalog taxonomy (categories inherit this property from parent catalog or category, products inherit this property from parent category)
```
Comment (yecli) [inline README.md]: (categories inherit this property from parent catalog or category, products inherit this property from parent category)
It is actual only for DownTree propagation. Need to generalize that it would be actual for UpTree.

File: README.md [hunk 3]
```diff
@@ -1,55 +1,129 @@
...
 
-<details><summary>Managing category and product tags</summary><p>
+## Scenarios
 
-![category-product-tags](https://user-images.githubusercontent.com/7536694/31497224-b5e4b1a8-af5e-11e7-90a1-918f120efddd.gif) </p></details>
+### Create New User Group
 
-#### Important! Don't forget to rebuild search indexes for categories and products after tagging process.
+1. Go to More->Contacts module-> select a user from the list and open user's details;
+1. Click the 'Edit' of the User Grpups;
```
Comment (yecli) [inline README.md]: Grpups
Typo

File: README.md [hunk 4]
```diff
@@ -1,55 +1,129 @@
...
+1. In the opened 'Member groups dictionary' enter the new User Group name and click the 'Add' button;
+1. Save the new user group;
+1. The new user group is now available for selection.
+
+![Add user group](docs/media/screen-add-usergroup.png
+)
+
+![User group added](docs/media/screen-usergroup-added.png)
+
+![User group available](docs/media/screen-usergroup-available.png)
```
Comment (yecli) [inline README.md]: Huge pictures, can you make them in 100% scale?

File: README.md [hunk 5]
```diff
@@ -1,55 +1,129 @@
...
+### Assign User Group to Product Category
+
+1. Go to More->Catalog Module->select a Product Category and click 'Manage';
+1. On Category details blade select the 'Tags' widget;
+1. Under 'Available tags' select the tag to assign to the Category;
+1. Click 'Add';
+1. Save the changes;
+1. Open the Category;
+1. Select a product and click the 'Tags' widget;
+1. The tag assigned to the Category will be displayed under the 'Inherited tag'. All Category products will inherit the tag assigned to the Category.
```
Comment (yecli) [inline README.md]: In case of DownTree propagation

File: README.md [hunk 6]
```diff
@@ -1,55 +1,129 @@
...
+1. Save the changes.
+
+![Assign user group to pricelist](docs/media/screen-assign-usergroup-to-pricelist.png)
+
+### Assign User Group to Marketing Promotion
+
+1. Go to More-Marketing module->Promotions;
+1. Select a Promotion and open the promotion details blade;
+1. Under 'Add user group' select 'User group contains';
+1. Enter the user group name into the correspondent field;
```
Comment (yecli) [inline README.md]: correspondent -> 'corresponding'?

File: README.md [hunk 7]
```diff
@@ -1,55 +1,129 @@
...
+
+![Assign tag to category](docs/media/screen-assign-tags.png)
+
+![Inherit tags](docs/media/screen-inherited-tags.png)
+
+### Assign User Group to a Price List Assignment
+
+1. Go to More->Pricing Module->select a Price list Assignment and open the Price list Assignments details;
+1. Under 'Add condition' select 'Shopper profile' and then 'User Group contain';
+1. Enter the User Group name into the correspondent field;
```
Comment (yecli) [inline README.md]: correspondent -> 'corresponding'?

File: README.md [hunk 8]
```diff
@@ -1,55 +1,129 @@
...
 <details><summary>Rebuild indexes</summary><p>
 
 ![indexation](https://user-images.githubusercontent.com/7536694/31497248-d12a5efe-af5e-11e7-8938-b39d2b03ad1e.gif) </p></details>
 
-<details><summary>Managing user groups</summary><p>
+### Tags Propagation Policies
+
+#### DownTree propagation policy
+
+When down-tree propagation policy is selected, all products inherit the Tag assigned to the Category.
```
Comment (yecli) [inline README.md]: assigned to the Category -> assigned to the parent categories

File: README.md [hunk 9]
```diff
@@ -1,55 +1,129 @@
...
+1. Restart the VC platform.
 
-![user-groups](https://user-images.githubusercontent.com/7536694/31497349-252148ce-af5f-11e7-9c4b-a2c9545fd853.gif) </p></details>
+![Settings](docs/media/screen-settings.png)
 
-##### As the result, products are displayed to the user only from the category that belongs to his `User group`.
+#### UpTree Policy Propagation
 
-<details><summary>Result view</summary><p>
+When the up-tree policy propagation is applied, the Product category will inherit the Product tag, i.e. if a tag assigned to a specific product in the Category, it will be automatically assigned to the Product Category as well.
```
Comment (yecli) [inline README.md]: it will be automatically assigned -> it will be propagated

File: README.md [hunk 10]
```diff
@@ -1,55 +1,129 @@
...
 
-![result](https://user-images.githubusercontent.com/7536694/31497775-53d994a4-af60-11e7-8544-90ccdf32f82a.gif) </p></details>
+In order to apply this type of propagation policy to Catalog categories and products, do the following:
 
+1. Go to Settings->Personalization->General;
+1. Under 'Selected policy for tags propagation' select 'UpTree';
+1. Restart the VC platform.
```
Comment (yecli) [inline README.md]: Do we need to duplicate these actions? Maybe it'd be better to write how to set policy and write what every policy means.

File: README.md [hunk 11]
```diff
@@ -1,55 +1,129 @@
...
+
+![Settings](docs/media/screen-settings1.png)
+
+#### Synchronize outlines
+
+The 'Synchronize outlines' function works for UpTree policy propagation and helps assign tags automatically from product to all product parents. The synchronization is based on the scheduled job.
```
Comment (yecli) [inline README.md]: helps assign tags automatically from product to all product parents ->
helps propagate tags from product to its parents

Note:
'Assign' is different from 'propagate' or 'inherit' on our terminology - we are not assigning tags to category in UpTree progagation, Tags are inherited

File: README.md (hunk 4 above)
Comment (mvktsk) [reply README.md]: Place pictures in the list under the corresponding item

File: README.md (hunk 2 above)
Comment (mayaalsamarrai) [reply README.md]: Do you mean that we should mention that this is DownTree propagation? What do you mean by Need to generalize that it would be actual for UpTree.?

File: README.md (hunk 4 above)
Comment (mayaalsamarrai) [reply README.md]: done

File: README.md (hunk 11 above)
Comment (mayaalsamarrai) [reply README.md]: ok, got it

File: README.md [hunk 12]
```diff
@@ -84,27 +84,19 @@ The Contacts should be first included into a specific User Groups and then the c
 
 ### Tags Propagation Policies
 
-#### DownTree propagation policy
+When down-tree propagation policy is selected, all products inherit the Tag propagated to the parent categories.
 
-When down-tree propagation policy is selected, all products inherit the Tag assigned to the parent categories.
-In order to apply this type of propagation policy to Catalog categories and products, do the following:
+When the up-tree policy propagation is applied, the Product category will inherit the Product tag, i.e. if a tag propagated to a specific product in the Category, it will be propagated to the Product Category as well.
```
Comment (yecli) [inline README.md]: Here "assigned" is correct:
 i.e. if a tag **assigned** to a specific product in the Category, it will be propagated to the Product Category as well.

Comment (mayaalsamarrai) [reply README.md]: ок!
